### ISSUE TITLE
Is this project discontinued or moved/forked?

### ISSUE THREAD

**Issue by ghost** [chjj/blessed#293]
I see 29 open pull requests and 124 open issues here on GitHub. There were no new commits on the code since more than 2 year. Is this project still maintained?

**Comment by ghost** [comment]
I have seen that there is a similar project named "blessed-contrib" but it is not a direct fork of the latest commit of this project: https://github.com/yaronn/blessed-contrib

Is "blessed-contrib" a full replacement of "blessed"? Does "blessed-contrib" contain the fixes and changes of the last commits of "blessed"?

**Comment by pt300** [comment]
I'm not sure if this project was definitely abandoned, but if you closely examine commits, you will find out last one isn't yet 2 years old.
About your other questions, I don't think it's the right place to ask them.

**Comment by ghost** [comment]
"blessed-contrib" is not a fork, it's an extension. So the main question here is still if the main project "blessed" is maintained by somebody.

**Comment by interfect** [comment]
I believe the project is now being maintained as https://github.com/blessed-ng/blessed, by the people who controll http://blessedjs.org.

**Comment by curioussavage** [comment]
Yeah Jesus forked it and I bought the domain. Have been meaning to work on documentation improvements but I have been swamped at work.

I'm tempted to try to start porting it to nim too...

**Comment by keevitaja** [comment]
@curioussavage you're still maintaining it or gave up? 

i am asking just cause i am building a project that depends on blessed.

**Comment by curioussavage** [comment]
Basically no not right now. I am still swamped at work and haven't had the time to put into this. I think the guy who forked it had some somewhat ambitious plans. I would just like to see some fixes for issues reported here and the pull requests that are good merged. If I find the time porting it to  a compiled lang is attractive to me so apps don't require a runtime to be installed.

**Comment by laurent22** [comment]
How about trying to recruit developers from blessed-contrib? It's still under development, has even more stars than blessed and is entirely based on it. Perhaps developers from this project would be interested in maintaining blessed too?

**Comment by rattrayalex** [comment]
I filed an issue suggesting that blessed-contribu fork & maintain: https://github.com/yaronn/blessed-contrib/issues/128

**Comment by curioussavage** [comment]
@laurent22 as was mentioned above blessed-contrib is a lib that adds more widgets. Its not a fork or anything but if people still work on it, it would be great if they could help out. I have not had the time or energy to contribute to the fork @piranna made. If he is still responsive I think it would be best if some interested people asked to take over. 

Personally I'm more interested in porting blessed to a compiled language these days.
